i18n/GLOSSARY_pt.md

```diff
@@ -19,6 +19,13 @@ Este glossário acompanha os [Princípios de GitOps](./PRINCIPLES_pt.md) e outro
 
     Quando o estado atual de um sistema se moveu ou está em processo de se afastar do [estado desejado](#estado-desejado), isso geralmente é chamado de desvio.
 
+- ## Obter    
```

Comment:
## Pull

Reply:
In the English version there's a correlation between then glossary term "Pull" and the principle "Pulled Automatically". We should stick to this pattern in the pt version, too.

So either "Obter", "Extrair" or "Puxar" (depending on what we land on for the principle's name) would be a better fit.

Reply:
I switched to Puxar